local/config.properties

```diff
@@ -21,3 +21,6 @@ apprunner.keymanager.password=password
 # Timeouts in millis for the proxied requests to the apps - if an app exceeds these when responding a 504 is returned
 apprunner.proxy.total.timeout=60000
 apprunner.proxy.idle.timeout=30000
+
+# Clean up rust intermediate build files
```

Comment:
Is there any reason to not enable this? Maybe simpler just to always remove build files.